messages.c

```diff
@@ -137,11 +137,7 @@ void messages_draw(MESSAGES *m, int x, int y, int width, int height)
                 h2 = STRING_IDX_MAX;
             }
 
-            if(msg->author) {
-                setcolor(COLOR_MAIN_SUBTEXT);
-            } else {
-                setcolor(COLOR_MAIN_TEXT);
-            }
+            setcolor(COLOR_MAIN_CHATTEXT);
```

Comment:
That's how chat text looked like before the merge. If it bothers anyone I can restore it.